{"uuid": "32a5d97c-8601-463c-a75b-1b97692667a0", "children": ["0d6973af-cbf4-4f3e-818c-c9eab69f3bef"], "befores": [{"name": "create_user", "status": "broken", "statusDetails": {"message": "KeyError: 'accessToken'\n", "trace": "  File \"C:\\cygwin64\\home\\Maxim\\Diplom\\Diplom_2\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\cygwin64\\home\\Maxim\\Diplom\\Diplom_2\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\cygwin64\\home\\Maxim\\Diplom\\Diplom_2\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\cygwin64\\home\\Maxim\\Diplom\\Diplom_2\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1139, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\cygwin64\\home\\Maxim\\Diplom\\Diplom_2\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 890, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\cygwin64\\home\\Maxim\\Diplom\\Diplom_2\\conftest.py\", line 16, in create_user\n    token = response.json()[\"accessToken\"]\n            ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^\n"}, "start": 1721746393735, "stop": 1721746393871}], "start": 1721746393735, "stop": 1721746393880}